resource/roboy_to_human_linkname_map.yaml

```diff
@@ -0,0 +1,60 @@
+---
+roboyToHumanLinkNameMap:
+  torso: chest
+  shoulder_right_link1: right_shoulder
```

Comment:
no, shoulder_right_link* points at the tiny ball on top of the roboy arm - this is definitely not matching to the left shoulder of the human... 
shoulder of the human corresponds to the upper arm of the robot, as you can see on the pic I posted in the chat